dockerfiles/Dockerfile.build

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:11
+LABEL maintainer="Digirati <opensource@digirati.com>"
+
+USER root
+
+RUN apt-get clean && apt-get update && apt-get -y upgrade \
+    && apt-get install -y python python-pip software-properties-common lsb-release apt-transport-https \
+    && curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable" \
+    && apt-get update && apt-get -y install docker-ce
+
+RUN pip install pre-commit
```

Comment:
We should target the specific version of `pre-commit` we want to install, this can be found using `pip search <package-name>`.

Also, I suggest rolling this `pip install` into the above `RUN` to keep the build image size down.